client/packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEdit.tsx

```diff
@@ -32,28 +32,44 @@ import {
   getAllocatedQuantity,
   getAllocatedPacks,
 } from './utils';
-import { useOutbound } from '../../api';
+import { DraftItem, useOutbound } from '../../api';
 import { DraftOutboundLine } from '../../../types';
 
 interface ItemDetailsModalProps {
   isOpen: boolean;
   onClose: () => void;
-  item: ItemRowFragment | null;
+  draft: DraftItem | null;
   mode: ModalMode | null;
 }
 
+const useFocusNumberOfPacksInput = (draft: DraftItem | null) => {
+  const batch = draft?.barcode?.batch;
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (!batch) return;
+      const input = document.getElementById(`pack_quantity_${batch}`);
+      if (input) {
+        input.focus();
+      }
+    }, 500);
+  }, [batch]);
+};
+
 export const OutboundLineEdit: React.FC<ItemDetailsModalProps> = ({
   isOpen,
   onClose,
-  item,
+  draft,
   mode,
 }) => {
+  const item = !draft ? null : draft.item ?? null;
```

Comment:
as per earlier comment, type safety breaks here, item can just be {id: string} due to `as` cast